database/migrations/2020_12_04_101640_add_phone_to_users_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddPhoneToUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->string('phone')->nullable()->after('password');
```

Comment:
Giả sử hệ thống này cho phép đăng nhập bằng phone nữa, thì chú sẽ define cột này như thế nào ?

Reply:
Em cho như này đc ko ạ
```suggestion
            $table->string('phone')->unique()->after('username');
```